src/iuse_actor.h

```diff
@@ -56,6 +56,9 @@ class iuse_transform : public iuse_actor
         /** if set transform item to container and place new item (of type @ref target) inside */
         itype_id container;
 
+        /** if set choose this variant when transforming */
+        std::string variant_type;
```

Comment:
One last thing! It looks like a variant should always be rolled (since one is rolled on save/load if it lacks one).
```suggestion
        std::string variant_type = "<any>";
```